src/common/services/base.service.ts

```diff
@@ -0,0 +1,21 @@
+export class BaseService {
+  static organizationId: string;
```

Comment:
lets reuse UserAccountService file to store this static field: orgId,...
and when we get ids from the db lets save these static properties in the class